### ISSUE TITLE
Transition to npm

### ISSUE THREAD

**Issue by nickredmark** [nicolaslopezj/roles#23]
Are there any plans to transition this package to npm?

**Comment by nicolaslopezj** [comment]
Yes! I'm waiting that Meteor take the first step of the transition and publish some core packages to npm

**Comment by nicolaslopezj** [comment]
This issue will be closed when the transition is made
